### PR TITLE
Submission of Flutter Sample for GSOC

### PR DESCRIPTION
**Pull request by nuelsoft**
Added Shine Flash to samples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
 
 ## Flutter Samples
 
+
```
Comment (divyanshub024) [inline README.md]: Please remove this extra line

**Review by divyanshub024 — CHANGES_REQUESTED** [review]
Please remove art/WhatsApp Video 2020-03-24 at 18.23.13.mp4

File: README.md (hunk 1 above)
Comment (nuelsoft) [reply README.md]: Done Sir. Deleted the mp4.

**Review by divyanshub024 — APPROVED** [review]
Thanks a lot for your contribution! 🙏

**Pull request merged**